doc/source/tune/faq.rst

```diff
@@ -773,4 +773,37 @@ API should be used to get the path for saving trial-specific outputs.
     The `TUNE_ORIG_WORKING_DIR` environment variable was the original workaround for
     accessing paths relative to the original working directory. This environment
     variable is deprecated, and the `chdir_to_trial_dir` flag described above should be
-    used instead.
\ No newline at end of file
+    used instead.
+
+
+.. _tune-multi-tenancy:
+
+How can I run multiple Ray Tune jobs on the same cluster at the same time (multi tenancy)?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Running multiple Ray Tune runs on the same cluster at the same
+time is not officially supported. We do not test this workflow and we recommend
+using a separate cluster for each tuning job.
+
+The reasons for this are:
+
+1. When multiple Ray Tune jobs run at the same time, they compete for resources.
+   One job could run all its trials at the same time, while the other job waits
+   for a long time until it gets resources to run the first trial.
+2. If it is easy to start a new Ray cluster on your infrastructure, there is often
```

Comment:
```suggestion
2. If starting a new Ray cluster on your infrastructure is easy, there is often
```

Reply:
I think this changes the meaning - if starting many clusters is easy, that's what they should do!

Reply:
Good catch. I do my best to understand the text and sometimes I mangle the meaning. Thanks for your patience!